src/battle_script_commands.c

```diff
@@ -1586,11 +1586,14 @@ static bool32 AccuracyCalcHelper(u16 move)
 
     if (WEATHER_HAS_EFFECT)
     {
-        if ((IsBattlerWeatherAffected(gBattlerTarget, B_WEATHER_RAIN) && (gBattleMoves[move].effect == EFFECT_THUNDER || gBattleMoves[move].effect == EFFECT_HURRICANE)))
-        {
-            // thunder/hurricane ignore acc checks in rain unless target is holding utility umbrella
-            JumpIfMoveFailed(7, move);
-            return TRUE;
+        if (IsBattlerWeatherAffected(gBattlerTarget, B_WEATHER_RAIN)){
+            if ((gBattleMoves[move].effect == EFFECT_THUNDER || gBattleMoves[move].effect == EFFECT_HURRICANE) || 
+                (move == MOVE_BLEAKWIND_STORM || move == MOVE_WILDBOLT_STORM || move == MOVE_SANDSEAR_STORM))
```

Comment:
The current code style in the expansion (outside of tests) are brackets on a new line. Also it should suffice to append all the storm moves after Hurricane. I don't think a new `if` is needed in this case.

Reply:
New 'if' is not required, I mostly did it this way for readability but if it goes against coding styles / performance issues I'm happy for it to be changed

Reply:
You could move all the checks after Rain to a new line. That would improve readability a bit. Though to be fair there is not a lot of consistency when it comes to long if statements in the repo but usually additional `if` for readability aren't used.

Reply:
No worries, cheers for that - I've made a new commit which should have the correct formatting

Reply:
thx